src/components/UI/SelectTokenRow/SelectTokenRow.js

```diff
@@ -23,17 +23,19 @@ export const SelectTokenRow = ({tokenData, onClick}) => {
             <div className={styles.name}>{name}</div>
           </div>
         </div>
-        <div className={styles.right}>
-          <>
-            {!isLoading ? (
-              <div className={styles.balance}>
-                {formatBalance(balance)} {symbol}
-              </div>
-            ) : (
-              <Loading size={LoadingSize.SMALL} />
-            )}
-          </>
-        </div>
+        {balance && (
```

Comment:
This isn't right.
Something fetching is failed and then `balance=null`.
More then that, in case `balance=null|undefiend` and `isLoading=true` we should display loading spinner.
Try to think on the condition that balance never set (i.e. undefined) and loading is false.